How to see from front end?

edengracee opened the pull request:
I'd like to see our progress so far and if we've done things correctly from the front end, so how should I visit to check and see? If it is by visiting TheSeniorExperience.github.io , site I keep getting error messages and am unable to see anything. If I should be trying another site or method, please let me know, thank you.

nyanford committed:
Update index.html

I added the hyperlinks that way others can get to the other pages. I would recommend. editing so it flows with how you want your webpage to look

edengracee committed:
Add files via upload

edengracee committed:
Merge pull request #1 from nyanford/nyanford-patch-1

Update index.html

edengracee committed:
index.html

edengracee committed:
Add files via upload

nyanford commented:
In order to see it on the front end, I have been downloading the file and looking at it through there. Additionally, I downloaded the Desktop version so I have been able to have one side with the front end ( from the link) and then another tab where I am typing the code on the terminal. I just keep refreshing and it shows the changes as I code

WillHaney committed:
things_to_do_cstat.html

edengracee committed:
index.html

nyanford committed:
Update home

WillHaney committed:
index.html

nyanford committed:
Update roommates.html

I added in a maroon background, the hyperlinks to the other  pages, a title, and your name so it would flow with the theme of the other pages

nyanford committed:
Update tamu.html

I added a maroon background, hyperlinks to all the pages, your name, and titles so it flows with the rest of the pags.

nyanford committed:
Update roommates.html

same changes I said before, just adding them to both files

nyanford committed:
Update blank.yml

I added maroon background, hyperlinks to the other pages, and it showing the recommendations were by you. I was not sure what your topic was so I just made the title and the name of the heading for the hyperlink "your title". Feel free to go through and replace it with your title name

edengracee committed:
Update roommates.html

edengracee committed:
Update roommates.html

kmanthei committed:
Create dining.html

kmanthei committed:
Update index.html

changing name for hyperlink

kmanthei committed:
Update dining.html

nyanford committed:
Update home.html

fixed hyperlinks

nyanford committed:
Update tamu.html

hyperlinked the files

nyanford committed:
Update roommates.html

nothing

nyanford committed:
Update tamu.html

added home hyperlink

nyanford committed:
Delete home

do not need it

nyanford committed:
Delete .github/TheSeniorExperience.github.io directory

about to reeupload as just files

nyanford committed:
Delete .github/workflows/blank.yml

not needed

nyanford committed:
Add files via upload

nyanford committed:
Update home.html

fixed all the hyperlinks

nyanford committed:
Update dining.html

fixed hyperlinks

nyanford committed:
Delete index.html

not needed

nyanford committed:
Update irene.html

removed photos and fixed hyperlinks

nyanford committed:
Update roommates.html

fixed hyperlinks

nyanford committed:
Update tamu.html

fixed hyperelinks

nyanford committed:
Update home.html

nyanford committed:
Update irene.html

labels

kmanthei committed:
Update roommates.html

Removed line 44 recommendations by just to see if this is causing the code not to run, I will replace once I see I is or isn't the problem. Just trouble shooting.

kmanthei committed:
Update roommates.html

Putting line 44 back because it wasn't the problem
